package dao;

import java.sql.Connection;
import java.sql.SQLException;

import javax.naming.InitialContext;
import javax.naming.NamingException;
import javax.sql.DataSource;

public class SedeDao {

	public static void insertSede (String nomeSede, String indirizzo, String citta) {
		
		try {
			Connection con = ((DataSource) new InitialContext().lookup("java:jboss/datasources/corsi")).getConnection();

			java.sql.PreparedStatement preparedStatement = null;

			String insertTableSQL = "INSERT INTO sede (nomeSede,indirizzo,citta) VALUES (?,?,?)";

			preparedStatement = con.prepareStatement(insertTableSQL);

			preparedStatement.setString(1, nomeSede);
			preparedStatement.setString(1, indirizzo);
			preparedStatement.setString(1, citta);

			preparedStatement.executeUpdate();

		} catch (SQLException | NamingException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		
	}

	public static void deleteSede (String nomeSede) {
		
		try {
		Connection con = ((DataSource) new InitialContext().lookup("java:jboss/datasources/corsi")).getConnection();

		java.sql.PreparedStatement preparedStatement = null;

		String insertTableSQL = "DELETE FROM sede WHERE nomeSede = ?";

		preparedStatement = con.prepareStatement(insertTableSQL);

		preparedStatement.setString(1, nomeSede);

		preparedStatement.executeUpdate();
	} catch (SQLException | NamingException e) {
		// TODO Auto-generated catch block
		e.printStackTrace();
	}
	}

	public static void updateSede (String nomeSede, String indirizzo, String citta, int id) {
		
		
		
	}







}
